src/main/webapp/js/instructorStudentList.js

```diff
@@ -24,13 +24,46 @@ $(document).ready(function() {
 
     // Binding for 'Show Emails' check box.
     $('#show_email').on('change', function() {
+        var $copyEmailButton = $('#copy_email_button');
+        var $emails = $('#emails');
+        
         if (this.checked) {
-            $('#emails').show();
+            $emails.show();
+            $copyEmailButton.show();
         } else {
-            $('#emails').hide();
+            $emails.hide();
+            $copyEmailButton.hide();
         }
+        
         filterEmails();
     });
+    
+    // Binding for copy email button
+    $('#copy_email_button').click(function(e) {
+        e.preventDefault();
+        var $copyEmailButton = $('#copy_email_button');
+        var tips = 'Emails now are selected, you can use <kbd>Ctrl + C</kbd> to COPY and <kbd>Ctrl + V</kbd> to PASTE'
+                    + ' to your email client. <br>'
+                    + '<small class="text-muted">This message will disappear in 5 seconds<small>';
```

Comment:
Hmm I see, that could be a PR worthy of its own then 😛 . I looked at the [browser compatibility](https://developer.mozilla.org/en-US/docs/Web/API/Document/execCommand) and it works as far back as IE9, which is the furthest version of IE we support, so I don't see a problem there.